pages/flow.js

```diff
@@ -1,15 +1,180 @@
+import { experiences } from "@/experiences";
+import useLocalStorageState from "use-local-storage-state";
+import Guide from "@/components/Guide/Guide";
+import GuideAnimator from "@/components/GuideAnimator/GuideAnimator";
+import Slider from "@/components/Slider/Slider";
+import Animation from "@/components/Animation/Animation";
+import TagCloud from "@/components/TagCloud/TagCloud";
+import Page from "@/components/Page/Page";
+import FlowContainer from "@/components/FlowContainer/FlowContainer";
+import Navigation from "@/components/Navigation/Navigation";
+import NavButton from "@/components/NavButton/NavButton";
+
+export default function TestFlow() {
+  const router = useRouter();
+
+  // local storage state to save everything at the end
+  const [allEntries, setAllEntries] = useLocalStorageState("anonymous_moods", {
+    defaultValue: [],
+  });
+
+  // state to hold the selection of the first tag cloud
+  const [experience, setExperience] = useState([]);
+
+  // state of the slider value
+  const [sliderValue, setSliderValue] = useState(0.5);
+
+  // state to hold the selection of the second tag cloud
+  const [reactions, setReactions] = useState([]);
+
+  // state that holds the color that is selected in the first tag cloud
+  const [color, setColor] = useState("grey");
+
+  // state that hold the current page
+  const [page, setPage] = useState(0);
+
+  const guides = [
+    "share your emotions ...",
+    "breathe",
+    "think about your day",
+    "how do you feel?",
+    "what do you feel right now?",
+    "how intense is this feeling for you?",
+    "what's your reaction?",
+  ];
+
+  function PageDisplay() {
+    switch (page) {
+      case 0:
+        return (
+          <Page>
+            <Guide bigger={true} text={"komm zur Ruh"} />
+          </Page>
+        );
+      case 1:
+        return (
+          <Page>
+            <GuideAnimator guides={guides} />
+          </Page>
+        );
+      case 2:
+        return (
+          <Page>
+            <Guide text={guides[4]} />
+            <TagCloud
+              tags={experiences}
+              colorSelected={true}
+              allowMultiple={false}
+              onSelectTag={handleSelectExperience}
+              selectedTags={experience}
+            />
+          </Page>
+        );
+      case 3:
+        return (
+          <Page>
+            <Guide text={guides[5]} />
+            <Slider
+              experience={experience}
+              sliderValue={sliderValue}
+              handleSliderChange={handleSliderChange}
+            />
+          </Page>
+        );
+      case 4:
+        return (
+          <Page>
+            <Guide text={guides[6]} />
+            <TagCloud
+              selectedTags={reactions}
+              tags={experience[0].reactions}
+              colorSelected={true}
+              allowMultiple={true}
+              onSelectTag={handleSelectReactions}
+            />
+          </Page>
+        );
+    }
+  }
```

Comment:
You should move this to its own file and treat it as a normal component. Which it is, since it is a function that returns JSX:

```suggestion
 function PageDisplay({
  guides,
  handleSelectExperience,
  experience,
  sliderValue,
  experiences,
  handleSliderChange,
  page,
  handleSelectReactions,
  reactions,
}) {
  switch (page) {
    case 0:
      return <Guide bigger={true} text={"komm zur Ruh"} />;
    case 1:
      return <GuideAnimator guides={guides} />;
    case 2:
      return (
        <>
          <Guide text={guides[4]} />
          <TagCloud
            tags={experiences}
            colorSelected={true}
            allowMultiple={false}
            onSelectTag={handleSelectExperience}
            selectedTags={experience}
          />
        </>
      );
    case 3:
      return (
        <>
          <Guide text={guides[5]} />
          <Slider
            experience={experience}
            sliderValue={sliderValue}
            handleSliderChange={handleSliderChange}
          />
        </>
      );
    case 4:
      return (
        <>
          <Guide text={guides[6]} />
          <TagCloud
            selectedTags={reactions}
            tags={experience[0].reactions}
            colorSelected={true}
            allowMultiple={true}
            onSelectTag={handleSelectReactions}
          />
        </>
      );
    default:
      return null;
  }
}
```

and a switch case without a default looks odd, therefore I added a default case.